### PR TITLE
added google_authentication role

### PR DESCRIPTION
**Pull request by santos-lucas**
Just adding First version of google_authenticator role.

### CODE REVIEW

File: collections/security/roles/google_authenticator/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+This role simply install google-authenticator tool and configure the basics to enable 2fa over ssh.
+
+## Instructions
+
+For now the Role has two possible configuration modes:
+
+1 - public key + google authenticator ( recomended )
+
+```yaml
+mfa_mode: password
```
Comment (oxedions) [inline collections/security/roles/google_authenticator/README.md:14]: Shouldnt it be the opposite? So pubkey here ?

**Review by oxedions — CHANGES_REQUESTED** [review]

**Comment by oxedions** [comment]
Dear @santos-lucas 
Thank you very much for this new feature !! 😊 
A small comment, then everything else seems ok !

**Comment by santos-lucas** [comment]
Yes, you`re right, already changed it !

File: collections/security/roles/google_authenticator/README.md (hunk 1 above)
Comment (santos-lucas) [reply collections/security/roles/google_authenticator/README.md]: yesss

**Comment by oxedions** [comment]
Super, thanks a lot !
I will add it soon to the automated tests, I am working on that.
Merging 😉

**Review by oxedions — APPROVED** [review]

**Pull request merged**